profile_bluesky/startup/instrument/devices/ad_rigaku_detector.py

```diff
@@ -63,22 +60,6 @@ class RigakuUfxcDetectorCam(AD_AcquireDetectorCamBase, CamBase):
     pool_max_buffers = None
     EXT_TRIGGER = 0
 
-    def setup_modes(self, num_triggers):
```

Comment:
This method is _required_ for the `AD_Acquire()` plan to work.  If it is deleted, the base class will raise a `NotImplementedError()` exception.

Reply:
If there is no work to be done (from bluesky's point of view), then:

```python
        yield from bps.null()  # at least must yield *some* bluesky message
```

Reply:
Ah, there it is below.